src/chrome/content/rules/Kitsu.io.xml

```diff
@@ -0,0 +1,12 @@
+<!--
+	Not resolved:
+		- www.kitsu.io
+
+-->
+<ruleset name="Kitsu (partial)">
```

Comment:
Referig to your comment www does not exist, so the host is fully covered with those rules.